dask_sql/physical/rel/custom/create_model.py

```diff
@@ -125,10 +125,35 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
             raise ValueError("Parameters must include a 'model_class' parameter.")
 
         target_column = kwargs.pop("target_column", "")
-        wrap_predict = kwargs.pop("wrap_predict", False)
-        wrap_fit = kwargs.pop("wrap_fit", False)
+        wrap_predict = kwargs.pop("wrap_predict", None)
+        wrap_fit = kwargs.pop("wrap_fit", None)
         fit_kwargs = kwargs.pop("fit_kwargs", {})
 
+        try:
+            ModelClass = import_class(model_class)
+        except ImportError:
+            raise ValueError(
+                f"Can not import model {model_class}. Make sure you spelled it correctly and have installed all packages."
+            )
+
+        model = ModelClass(**kwargs)
+
+        if wrap_predict is None:
+            if "sklearn" in model_class or (
+                "cuml" in model_class and "cuml.dask" not in model_class
+            ):
+                wrap_predict = True
+            else:
+                wrap_predict = False
+        if wrap_fit is None:
+            if (
+                "sklearn" in model_class
+                or ("cuml" in model_class and "cuml.dask" not in model_class)
+            ) and hasattr(model, "partial_fit"):
+                wrap_fit = True
+            else:
+                wrap_fit = False
+
```

Comment:
Minor nit:
```suggestion
        if "sklearn" in model_class or (
            "cuml" in model_class and "cuml.dask" not in model_class
        ):
            wrap_predict = True if wrap_predict else False
            wrapt_fit = True if wrap_fit and hasattr(model, "partial_fit") else False
```

Reply:
I think we still need the outer conditions `if wrap_predict is None` and `if wrap_fit is None`, otherwise if the user specifies either of these parameters then their specification will be overwritten. Let me know what you think?

Reply:
Yeah you're right, we can scrap this suggestion.